sdk/metricsadvisor/perf-tests/ai-metrics-advisor/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "../../../../tsconfig.package",
+  "compilerOptions": {
+    "target": "es5",
+    "module": "commonjs",
+    "declarationDir": "./typings/latest",
+    "outDir": "./dist-esm",
+    "lib": ["dom", "es5", "es6", "es7", "esnext"]
```

Comment:
Looks like this has taken inspiration from storage perf tests which required special options.
Can you copy tsconfig in the Text analytics perf test project instead?

Reply:
Updated